idea_town/frontend/static-src/app/templates/experiment-page.js

```diff
@@ -20,16 +20,16 @@ export default `
 
       <div class="details-content">
         <div class="details-overview">
-          <img src="{{thumbnail}}" width="260">
-          <section>
-            <h3>Measurements</h3>
-            <p class="disclaimer">All data is collected anonymously and used only to help us improve this test. <a href="">Learn more.</a></p>
-            <ul class="measurement">
-              <li>Usage per session</li>
-              <li>Index of selected results</li>
-              <li>Time spent searching</li>
-            </ul>
-          </section>
+          <img src="{{thumbnail}}" width="260" height="auto">
+            <section>
+              <h3>Measurements</h3>
+              <p class="disclaimer">All data is collected anonymously and used only to help us improve this test. <a href="">Learn more.</a></p>
+              <ul class="measurement">
+                <li>Usage per session</li>
+                <li>Index of selected results</li>
+                <li>Time spent searching</li>
+              </ul>
+            </section>
```

Comment:
Nit: reduce indent here since it isn't nested within the `<img>` tag, plus it'll align with the following `<section>` tag.